p2p_wallet/Scenes/CreateOrRestoreWallet/Common/Pincode/Pincode/PincodeViewModel.swift

```diff
@@ -29,10 +38,12 @@ final class PincodeViewModel: BaseViewModel {
     let openMain = PassthroughSubject<(String, Bool), Never>()
 
     let isBackAvailable: Bool
+    // TODO: seems like it is used only in confirm state, move to case?
```

Comment:
Not sure. I have not seen it before. But I see how PincodeChangeCoordinator use it. I guess we can put isBackAvailable: Bool = true, successNotification: String into PincodeParameters or smth like that